js/src/routes.js

```diff
@@ -16,11 +16,15 @@
 
 import {
   Accounts, Account, Addresses, Address, Application,
-  Contract, Contracts, Dapp, Dapps,
+  Contract, Contracts, Dapp, Dapps, HistoryStore,
```

Comment:
Not 100% convinced on this location. It is only related to keeping a history of views, but I'm open to suggestions to other locations.